nome.py

```diff
@@ -0,0 +1,30 @@
+saldo = 0
```

Comment:
Recomendo usar nomes de variáveis sempre em inglês
```suggestion
balance = 0
```